monkey/monkey_island/cc/next_ui/src/app/consts.tsx

```diff
@@ -0,0 +1,6 @@
+function getJSRuntimePort() {
+    // TODO remove with the NEXT_JS_UI_FEATURE in monkey_island/cc/feature_flags.py:1
+    return process.env.NEXT_PUBLIC_JAVASCRIPT_RUNTIME_PORT || 5000;
```

Comment:
Can we do this now?

Reply:
Not yet, we'll do it once the old UI is removed